app/models/katello/glue/pulp/repos.rb

```diff
@@ -6,12 +6,23 @@ def self.included(base)
       base.send :include, InstanceMethods
     end
 
-    def self.repo_path_from_content_path(environment, content_path)
-      content_path = content_path.sub(/^\//, "")
-      path_prefix = [environment.organization.label, environment.label].join("/")
+    def self.repo_path_from_org_lifecycle_content_path(org_label, lifecycle_label, content_path)
+      content_path.sub!(%r|^/|, '')
```

Comment:
I don't think modifying a string coming in as argument is a good idea. I would be surprised if I called the method, and getting the argument string actually modified as a side effect.

Reply:
I agree, sorry, forgot that it's "by reference".